### PR TITLE
Implement draw.go

### PR DESCRIPTION
**Pull request by enochtsang**

### CODE REVIEW

File: frontend/src/modules/scribble/components/Canvas.vue [hunk 1]
```diff
@@ -81,21 +83,28 @@ export default {
...
-      this.context.moveTo(from.x, from.y);
-      this.context.lineTo(to.x, to.y);
-      this.context.strokeStyle = brushStyle.brushColor;
-      this.context.lineWidth = brushStyle.brushSize;
-      this.context.lineCap = "round";
-      this.context.stroke();
-      this.context.closePath();
-
-      EventBus.$emit("brushStroke", {
+      console.log("Sending Draw Action")
```
Comment (bseto) [inline frontend/src/modules/scribble/components/Canvas.vue:86]: can we get rid of the console log messages on line 86 and 98?

File: frontend/src/modules/scribble/components/Canvas.vue [hunk 2]
```diff
@@ -81,21 +83,28 @@ export default {
...
+      let drawActionMessage = createDrawActionMessage({
```
Comment (bseto) [inline frontend/src/modules/scribble/components/Canvas.vue:87]: niceeeeee

File: backend/game/scribble/handler/draw/draw.go [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+
+	reg.SendToCaller(clientID, historyBytes)
+}
+
+func (h *Handler) ForwardAction(
+	drawAction DrawAction,
+	caller identifier.Client,
+	registry registry.Registry,
+) {
+	log.Infof("Forwarding draw action: ", drawAction)
```
Comment (bseto) [inline backend/game/scribble/handler/draw/draw.go:143]: mmm I think if you want this logged, maybe log it as Debugf. I don't want to see this on the log for every user unless i'm in debug mode (although I don't have a debug flag yet in main xD)

Comment (enochtsang) [reply backend/game/scribble/handler/draw/draw.go:143]: I'll just take them off, I was kinda unsure if I should keep this or not.

**Pull request merged**